core/models/asset.py

```diff
@@ -231,6 +231,7 @@ class Asset(models.Model):
     serial_number = models.CharField(
         unique=True, null=True, blank=True, max_length=50)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    purchase_date = models.DateField()
```

Comment:
Considering the nature of the purchase date; It **cannot** be in the future...how about we include a validator method to the `purchase_date` model field to raise a _ValidationError_ when the date is set to a value in the future.
I do realise that validation that this would increase the scope of the task. What do you think @QUDUSKUNLE @egichuri ?

Reply:
since it's validation for a field added it can be done as part of this ticket

Reply:
@QUDUSKUNLE You can reference Django's [docs](https://docs.djangoproject.com/en/2.0/ref/validators/) to understand how to go about it.

Reply:
@collinmutembei Thanks. Changes effected.